G/GDAL/build_tarballs.jl

```diff
@@ -153,6 +156,7 @@ dependencies = [
     Dependency("NetCDF_jll"; compat="400.902.208", platforms=hdf5_platforms),
     Dependency("HDF5_jll"; compat="~1.14", platforms=hdf5_platforms),
     Dependency("Arrow_jll"; compat="10"),
+    Dependency("XML2_jll"),
```

Comment:
Untested, but something like this might work:
```suggestion
    Dependency("XML2_jll"),
    BuildDependency(PackageSpec(; name="OpenMPI_jll", version="4.1.6"); platforms=filter(p -> nbits(p)==32, platforms)),
```